apps/outlook-rooms-addin/src/app/rooms/features-filter.service.ts

```diff
@@ -1,30 +1,36 @@
-import { Injectable } from '@angular/core';
-import { BehaviorSubject, Observable, of } from 'rxjs';
-import { take, map, filter } from 'rxjs/operators';
+import { Injectable, OnDestroy, EventEmitter } from '@angular/core';
+import { BehaviorSubject, Observable, of, Subscription } from 'rxjs';
+import { take, map, filter, switchMap } from 'rxjs/operators';
 import { Space } from '@placeos/spaces';
 import { EventFormService } from '@placeos/events';
 
 @Injectable({
     providedIn: 'root',
 })
 export class FeaturesFilterService {
-    public spaces$ = this._state.available_spaces;
-    selected_features$: Observable<any>;
-    features_list: Array<string> = [];
-    filtered_spaces: Space[] = [];
-    _features: BehaviorSubject<any> = new BehaviorSubject<any>([]);
+    public spaces$: Observable<Space[]> = this._state.available_spaces;
+    public updated_spaces$: Observable<Space[]>;
+    public updated_spaces_emitter: EventEmitter<boolean> = new EventEmitter(
```

Comment:
`Emitters` should be used for `@Outputs` in components. Use a rxjs `Subject` instead.